tachyon/zk/lookup/compress_expression_unittest.cc

```diff
@@ -9,12 +9,10 @@
 #include "gtest/gtest.h"
 
 #include "tachyon/zk/expressions/expression_factory.h"
-#include "tachyon/zk/lookup/test/compress_expression_test_setting.h"
+#include "tachyon/zk/lookup/test/compress_expression_test.h"
 
 namespace tachyon::zk {
 
-class CompressExpressionTest : public CompressExpressionTestSetting {};
-
 TEST_F(CompressExpressionTest, CompressExpressions) {
```

Comment:
inherit `public CompressExpressionTest` and change the class name, or change `TEST_F` to `TEST`

Reply:
I think we don't have to inherit the class in this case.

Reply:
In that case, change `TEST_F` to `TEST`?

Reply:
This still needs a fixture, otherwise it won't get compield.